src/main/java/com/redhat/devtools/lsp4ij/LanguageServiceAccessor.java

```diff
@@ -371,9 +405,7 @@ public MatchedLanguageServerDefinitions getMatchedLanguageServerDefinitions(@Not
             languages.add(language);
         }
         FileType fileType = file.getFileType();
-        if (fileType != null) {
-            languages.add(fileType);
-        }
+        languages.add(fileType);

```

Comment:
maybe use a LinkedHashSet for languages

Reply:
why? It works great since the start of the project.